datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxUpgradeController.java

```diff
@@ -34,7 +34,7 @@
     @CheckPermissionByResourceName(action = AuthorizationResourceAction.UPGRADE_DATALAKE)
     public SdxUpgradeResponse upgradeClusterByName(@ResourceName String clusterName, SdxUpgradeRequest request) {
         String userCrn = ThreadBasedUserCrnProvider.getUserCrn();
-        lockComponentsIfRuntimeUpgradeIsDisabled(request, userCrn, clusterName);
+        //lockComponentsIfRuntimeUpgradeIsDisabled(request, userCrn, clusterName);
```

Comment:
commented out

Reply:
We should discuss if there is any case when we should update the lock components request parameter.